contentreview-impl/tool/src/java/org/sakaiproject/contentreview/servlet/SubmissionCallbackServlet.java

```diff
@@ -144,6 +155,21 @@ protected void doPostJSON(HttpServletRequest request, HttpServletResponse respon
         return;
     }
 	
+	public Map<String,Object> obtainGlobalTurnitinLTITool(String turnitinSite){
+		M_log.debug("Getting global TII LTI tool");
```

Comment:
Again, duplicating the code can easily lead to fixes only being done in some places down the line.